clang-formatting/filesWithFormattingErrors/goodFileInHere.c

```diff
@@ -0,0 +1,100 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+typedef struct DateAndTime
+{
+    uint64_t hour;
+    uint64_t minutes;
+    uint64_t seconds;
+    uint64_t msec;
+} DateAndTime;
```

Comment:
Should this be changed from DateAndTime? I do not see anything related to the date

Reply:
Entirely a fake C file that is just used to catch formatting errors, any content in the file itself related to code is irrelevant.